src/include/rbd/librbd.h

```diff
@@ -144,6 +144,8 @@ enum {
   RBD_IMAGE_OPTION_JOURNAL_ORDER = 5,
   RBD_IMAGE_OPTION_JOURNAL_SPLAY_WIDTH = 6,
   RBD_IMAGE_OPTION_JOURNAL_POOL = 7,
+  RBD_IMAGE_OPTION_FEATURES_SET = 8,
```

Comment:
@yangdongsheng Yes -- and then remove RBD_IMAGE_OPTION_FEATURES_SET since RBD_IMAGE_OPTION_FEATURES (or default features) will apply the optional RBD_IMAGE_OPTION_FEATURES_MASK